src/ameba/formatter/github_actions_formatter.cr

```diff
@@ -55,7 +55,11 @@ module Ameba::Formatter
         time_elapsed = Time.monotonic - started_at
       end
 
-      File.write(step_summary_file, summary(sources, time_elapsed))
+      summary = summary(sources, time_elapsed)
+      if (info = File.info?(step_summary_file)) && !info.size.zero?
+        summary = "\n\n#{summary}"
+      end
+      File.write(step_summary_file, summary, mode: "a")
```

Comment:
can you elaborate what use case we cover here? If this needed to allow running ameba multiple times in the same GHA step?

Reply:
I think I've been over-protective here, gonna remove that.